lib/package.py

```diff
@@ -89,19 +89,20 @@ def __repr__(self):
         return self.name
 
 
-    def download_files(self):
+    def download_files(self, recurse=True):
```

Comment:
The verb is form is ["recurse"](https://en.wiktionary.org/wiki/recurse). I can change to "recursive" or "recursively" if you prefer.